src/store/modules/permission.js

```diff
@@ -20,11 +20,12 @@ import { asyncRouterMap, constantRouterMap } from '@/config/router'
 function hasApi (apis, route) {
   if (route.meta && route.meta.permission) {
     for (const permission of route.meta.permission) {
-      if (apis.includes(permission)) {
-        return true
+      console.log(route.meta.name, permission)
```

Comment:
@shwstppr could this make permissions too strict; we've use this to show views when we want any of the APIs allowed in the list. Let's discuss tomorrow.